t/JrDevJobs.t

```diff
@@ -4,6 +4,22 @@ use warnings;
 use Test::More;
 use DDG::Test::Spice;
 
-use_ok('DDG::Spice::JrDevJobs');
+ddg_spice_test(
+    [
+        'DDG::Spice::JrDevJobs'
+    ],
+    'jrdevjobs ruby' => test_spice(
+        '/js/spice/jr_dev_jobs/ruby',
+        call_type => 'include',
+        caller => 'DDG::Spice::JrDevJobs',
+        is_cached => 1
+    ),
+    'web developer jrdevjobs' => test_spice(
+        '/js/spice/jr_dev_jobs/web%20',
```

Comment:
with `trim` you can drop the `%20` because there won't be any leading/trailing spaces 👍